scripts/zones/The_Eldieme_Necropolis_[S]/npcs/Erlene.lua

```diff
@@ -21,16 +21,16 @@ require("scripts/globals/status");
 function onTrade(player,npc,trade)
     local ALittleKnowledge = player:getQuestStatus(CRYSTAL_WAR, A_LITTLE_KNOWLEDGE);
     local ALittleKnowledgeProgress = player:getVar("ALittleKnowledge");
-	
+    
     if (ALittleKnowledge == QUEST_ACCEPTED and ALittleKnowledgeProgress == 1) then
         if (trade:hasItemQty(2550, 12) and trade:getGil() == 0 and trade:getItemCount() == 12) then
             if( player:getMainJob() == JOB_BLM or
                 player:getMainJob() == JOB_RDM or
                 player:getMainJob() == JOB_SMN or
                 player:getMainJob() == JOB_BLU) then
-                player:startEvent(12, 1);
+                player:startEvent(0x000C, 1);

```

Comment:
Just so you know you do not have to change these to hex. I personalty prefer we un-hex-ify things that do not explicitly require it (or that do not combine with other values such as a mask) just because people who aren't hex savy get confused by it (dsp's entire wiki shows event IDs in decimal..).